### PR TITLE
 VMwareVCenter

### PR DESCRIPTION
**Pull request by yossifirouz**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/VMwareVCenter/ModelingRules/VMwareVCenter/VMwareESXiModelingRules.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: VMware vCenter
```
Comment (meital-aizen) [inline Packs/VMwareVCenter/ModelingRules/VMwareVCenter/VMwareESXiModelingRules.yml:1]: Please change the name of the file to VMWareVCenter...

File: Packs/VMwareVCenter/ModelingRules/VMwareVCenter/VMwareVCenterModelingRules.xif [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+| filter _raw_log contains "vpxd[" or _raw_log contains "[vmafdd]" or _raw_log contains " vmdird "
+| alter Timestamp= arrayindex(regextract(_raw_log ,"(\d+\-\d+\-\d+\w\d+\:\d+\:\d+\.\d+\w+)") ,0)
+| alter Message_vpxd= arrayindex(regextract(_raw_log ,"vpxd\[\d+\](.*)") ,0)
+| alter user0_vpxd= arrayindex(regextract(_raw_log ,"user\s\<([^\>]+)") ,0)
+| alter user1_vpxd= arrayindex(regextract(_raw_log ,"User\:\s'([^'])") ,0)
+| alter Message_vmdrid= arrayindex(regextract(_raw_log ,"@\d+\:\s([^\t]+)") ,0)
+| alter Message_vmafdd= arrayindex(regextract(_raw_log ,"vmafdd.*\]\s[^\:]+\:\s([^\t]+)") ,0)
+| alter eventName_vmafdd = arrayindex(regextract(_raw_log ,"vmafdd.*\]\s([^\:]+)") ,0)
+| alter Message = coalesce(Message_vpxd ,Message_vmdrid ,Message_vmafdd )
+| alter user = coalesce(user0_vpxd , user1_vpxd )
```
Comment (evisochek) [inline Packs/VMwareVCenter/ModelingRules/VMwareVCenter/VMwareVCenterModelingRules.xif:27]: Where are the XDM fields that you mapped to?

File: Packs/VMwareVCenter/ModelingRules/VMwareVCenter/VMwareVCenterModelingRules.xif [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+| filter _raw_log contains "| vlsi |" or _raw_log contains "| vim" or _raw_log contains "jointool" or _raw_log contains "heard-" 
+| alter Timestamp0= arrayindex(regextract(_raw_log ,"(\d+\-\d+\-\d+\w\d+\:\d+\:\d+\.\d+\w+)") ,0)
+| alter Timestamp1 = arrayindex(regextract(_raw_log ,"\[(\d+\-\d+\-\d+\s\d+\:\d+\:\d+)") ,0)
+| alter Timestamp = coalesce(Timestamp0 ,Timestamp1 )
+| alter LogLevel = arrayindex(regextract(_raw_log ,"\d+\-\d+\-\d+\w\d+\:\d+\:\d+\.\d+\w+\s\|\s+(\S+)") ,0)
+| alter Message0 = arrayindex(regextract(_raw_log ,"\|\s\d+\s\|\s([^\t]+)") ,0)
+| alter Message1 = arrayindex(regextract(_raw_log ,"\]\s([^\t]+)") ,0)
+| alter Message = coalesce(Message0, Message1)
+
+| alter XDM.Auth.event_timestamp = Timestamp
```
Comment (evisochek) [inline Packs/VMwareVCenter/ModelingRules/VMwareVCenter/VMwareVCenterModelingRules.xif:11]: you are in Audit data model why you mapping to auth?

File: Packs/VMwareVCenter/ModelingRules/VMwareVCenter/VMwareVCenterModelingRules.xif [hunk 4]
```diff
@@ -0,0 +1,27 @@
+[Model: dataset ="VMware_vCenter_raw", moder=Audit]
```
Comment (evisochek) [inline Packs/VMwareVCenter/ModelingRules/VMwareVCenter/VMwareVCenterModelingRules.xif:1]: model not "moder"

File: Packs/VMwareVCenter/ModelingRules/VMwareVCenter/VMwareVCenterModelingRules.xif [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+| alter Message1 = arrayindex(regextract(_raw_log ,"\]\s([^\t]+)") ,0)
+| alter Message = coalesce(Message0, Message1)
+
+| alter XDM.Auth.event_timestamp = Timestamp
+, XDM.Auth.threat.description = Message
+, XDM.identity.name = LogLevel 
+
+
+
+[Model: dataset ="VMware_vCenter_raw", moder=Auth]
```
Comment (evisochek) [inline Packs/VMwareVCenter/ModelingRules/VMwareVCenter/VMwareVCenterModelingRules.xif:17]: model it with capital letters "MODEL"

**Pull request closed without merging**